lms/templates/index.html

```diff
@@ -18,7 +18,7 @@
               ## Translators: 'Open edX' is a brand, please keep this untranslated. See http://openedx.org for more information.
               <h2>${_("It works! This is the default homepage for this Open edX instance.")}</h2>
```

Comment:
@clytwynec Small nit, but can we change this to a ```<p>``` element.   It's rarely ever correct to put an ```<h2>``` directly below an ```<h1>``` or any subheading for that matter.  This just legitimizes it.  In this context specifically, the h2 text does not qualify as a heading.  It's one of those things where the dev used it to apply styling.  Applies to the stanford theme too. Otherwise, this is good to go.